polkadot-parachains/src/chain_spec_helpers.rs

```diff
@@ -48,21 +48,15 @@ pub struct WellKnownKeys;
 
 impl WellKnownKeys {
 	pub fn endowed() -> Vec<AccountId> {
-		Self::council()
+		vec![Alice.to_account_id(), Bob.to_account_id()]
```

Comment:
Can we not also endow `Charlie` so we can remove his fauceting to save some time in the `bootstrap_demo_community.py`? Fauceting is anyhow tested in the `bot-community.py`

Reply:
we could but I just wanted the config to match the encointer-node. I actually like that the bootstrap_demo_community tests as much basic stuff as possible already

Reply:
not entirely sure

Reply:
ok, will add Charlie ;-)